routes/movies.routes.js

```diff
@@ -0,0 +1,57 @@
+// starter code in both routes/celebrities.routes.js and routes/movies.routes.js
+const router = require("express").Router();
+const Movie = require("../models/Movie.model");
+const Celebrity = require("../models/Celebrity.model");
+
+// all your routes here
+router.get("/movies/create", async (req, res) => {
+  try {
+    const celebrities = await Celebrity.find();
+    res.render("movies/new-movie", { celebrities });
+  } catch (error) {
+    console.log(error);
+  }
+});
+
+router.post("/movies/create", async (req, res, next) => {
+  const { title, genre, plot, cast } = req.body;
+  try {
+    const movie = await Movie.create({
+      title,
+      genre,
+      plot,
+      cast,
+    });
+
+    res.redirect("/movies");
```

Comment:
You don't have the route `/movies` so every time you create a movie, it'll redirect you to a 404 page.